src/compiler/index.js

```diff
@@ -14,25 +14,31 @@
  * limitations under the License.
  */
 
+import * as compiler from '@ampproject/bento-compiler';
+
+import {getBuilders} from './builders';
+
 /**
- * Returns the HTML for an AMP Document with eligible components server-rendered.
- *
- * @param {string} html
- * @return {string}
+ * @typedef {*} TreeProto
+ * TODO: replace with definition in bento-compiler.
```

Comment:
self-nit: is it possible to use the JSDoc type in the bento-compiler repo?

Reply:
I don't see why not?

Reply:
Lets discuss offline